### PR TITLE
ISPN-14633 LPUSH and LPUSHX commands

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-14633

### CODE REVIEW

**Comment by wburns** [comment]
Is this a duplicate of https://github.com/infinispan/infinispan/pull/10909 ?

**Comment by karesti** [comment]
@jabolina @wburns this is ready for review

File: server/resp/src/test/java/org/infinispan/server/resp/RespListCommandsTest.java [hunk 1]
```diff
@@ -57,4 +57,46 @@ public void testRPUSHX() {
             .hasMessageContaining("ERRWRONGTYPE");
    }
 
+   public void testLPUSH() {
```
Comment (jabolina) [inline server/resp/src/test/java/org/infinispan/server/resp/RespListCommandsTest.java:60]: Should we verify the order, too?

File: server/resp/src/main/java/org/infinispan/server/resp/commands/list/internal/PUSH.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      EmbeddedMultimapListCache<byte[], byte[]> listMultimap = handler.getListMultimap();
+      AggregateCompletionStage<Void> aggregateCompletionStage = CompletionStages.aggregateCompletionStage();
+      // TODO: putAll operation on listMultimap ?
+      for (int i = 1; i < arguments.size(); i++) {
+         CompletionStage<Void> push = first ? listMultimap.offerFirst(key, arguments.get(i)) : listMultimap.offerLast(key, arguments.get(i));
+         aggregateCompletionStage.dependsOn(push);
+      }
+
+      return CompletionStages.handleAndCompose(aggregateCompletionStage.freeze(), (r, t) -> {
+         if (t instanceof CacheException) {
```
Comment (jabolina) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/list/internal/PUSH.java:58]: We are handling only `CacheException`, so we still execute the size operation if we have another type of exception here.

**Review by jabolina — COMMENTED** [review]
Just a couple of small things.

File: server/resp/src/test/java/org/infinispan/server/resp/RespListCommandsTest.java (hunk 1 above)
Comment (karesti) [reply server/resp/src/test/java/org/infinispan/server/resp/RespListCommandsTest.java:60]: not here. we need it in another PR with other operations

**Comment by karesti** [comment]
@jabolina I reported your changes

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Integrated into main, thanks @karesti!

**Pull request merged**